Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -495,6 +516,13 @@
     smokePrototype: AluminiumMetalFoam
   - type: StaticPrice
     price: 350
+  - type: CartridgeAmmo #Sunrise-Start
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)